examples/research_hub/README.rst

```diff
@@ -0,0 +1,28 @@
+********************************
+The Energy Wizard - Research Hub
+********************************
+
+This example demonstrates how to scrape publication abstracts and researcher profiles,
```

Comment:
Add a note that this is for the NREL researcher hub only (remember our repo is open source!)